website/docs/r/backup_plan.html.markdown

```diff
@@ -0,0 +1,57 @@
+---
+layout: "aws"
+page_title: "AWS: aws_backup_plan"
+sidebar_current: "docs-aws-resource-backup-plan"
+description: |-
+  Provides an AWS Backup plan resource.
+---
+
+# aws_backup_plan
+
+Provides an AWS Backup plan resource.
+
+## Example Usage
+
+```hcl
+resource "aws_backup_plan" "example" {
+	name = "tf_example_backup_plan"
+
+	rule {
+		rule_name 			= "tf_example_backup_rule"
```

Comment:
The equals sign alignment of the example here and some of the testing appears to be an issue. Mixed tabs and spaces?

Reply:
Yeah, this is due to me having different tab settings for Go and Terraform.